### DISCUSSION TITLE
What is the `jsx` attribute in TypeScript types?

### DISCUSSION THREAD

**Discussion by jaswrks** [preactjs/preact#4169] [General]
https://github.com/preactjs/preact/blob/e24fdad9d5e4459d7b461f3c807be049b9c509cf/src/index.d.ts#L57

Hi. May I please get some help understanding what this attribute is and where it comes from? I can't seem to find anything in the preact codebase that implements this, or considers it. Is this still relevant?

**Comment by andrewiggins** [comment]
Hmm I forget. I think it was added cuz some poplar library (maybe a styling library?) used it. In retrospect, the library should probably extend these types instead of us hardcoding them here.
